### PR TITLE
Updating Actions Overview and Donut

### PR DESCRIPTION
**Pull request by bmclaughlin**
Reorganized Overview page and updated donut chart per UXD mockups [1].  Addresses [2] and [3].

[1] https://redhat.invisionapp.com/share/V4NY3C6XKRW#/screens/350113377
[2] https://projects.engineering.redhat.com/browse/RHIADVISOR-472
[3] https://projects.engineering.redhat.com/browse/RHIADVISOR-473

screenshot

### CODE REVIEW

**Comment by AllenBW** [comment]
@katierik tagging yah in for [verification on the comment above](https://github.com/RedHatInsights/insights-advisor-frontend/pull/227#pullrequestreview-214025042)

File: src/PresentationalComponents/Charts/ActionsOverviewDonut.js [hunk 1]
```diff
@@ -10,7 +10,7 @@ class AdvisorOverviewDonut extends React.Component {
     getChart = (theme, typeNames) => <ChartDonut
         data={ this.props.category.map((value, key) => ({ x: typeNames[key], y: value, label: `${typeNames[key]}: ${value}` })) }
         theme={ theme }
-        height={ 200 }
+        height={ 200 }ChartTheme
```
Comment (AllenBW) [inline src/PresentationalComponents/Charts/ActionsOverviewDonut.js]: eekkkkk 😭

Comment (bmclaughlin) [reply src/PresentationalComponents/Charts/ActionsOverviewDonut.js]: I really hate the default 'middle click paste' option on mouse configurations...

**Comment by AllenBW** [comment]
Updated ss for posterity 
Screen Shot 2019-03-13 at 3 14 17 PM

File: src/PresentationalComponents/Charts/ActionsOverviewDonut.scss [hunk 2]
```diff
@@ -1,5 +1,20 @@
 .chart-container{
+  height: 200px;
+  width: 200px;
   .VictoryContainer {
     height: 200px !important;
```
Comment (AllenBW) [inline src/PresentationalComponents/Charts/ActionsOverviewDonut.scss]: good news we can get rid of this without impact!
```suggestion
```

File: src/PresentationalComponents/Charts/ActionsOverviewDonut.scss [hunk 3]
```diff
@@ -1,5 +1,20 @@
 .chart-container{
+  height: 200px;
+  width: 200px;
```
Comment (AllenBW) [inline src/PresentationalComponents/Charts/ActionsOverviewDonut.scss]: also don't need to set the width cuz we're already setting the height
```suggestion
```

File: src/PresentationalComponents/Charts/ActionsOverviewDonut.js [hunk 4]
```diff
@@ -85,41 +86,51 @@ class AdvisorOverviewDonut extends React.Component {
...
     render () {
+        const { className } = this.props;
         const label = <svg
             className="chart-label"
             height={ 1 }
         >
             <ChartLabel
-                style={ { fontSize: 20 } }
-                text={ this.props.category.length ? `${this.props.category.reduce((sum, curr) => sum + curr)} Issues` : '' }
+                style={ { fontSize: 25 } }
```
Comment (AllenBW) [inline src/PresentationalComponents/Charts/ActionsOverviewDonut.js]: [should probably keep font size at 20 ](https://github.com/patternfly/patternfly-react/blob/master/packages/patternfly-4/react-charts/src/components/DonutChart/examples/SimpleChart.js#L32)

**Pull request merged**